CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,30 @@
+Code of Conduct
+===============
+
+Like the technical community as a whole, the Python-based SVG editor team and community is made up of a mixture of professionals and volunteers from all over the world.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to when they're participating within this community and project. These rules apply equally to founders, mentors and those seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us, the technical community and the conferences and usergroups we hope to guide new speakers to.
```

Comment:
since we don't organise any conference and user groups, we could just remove the parts that mention them